airflow_connector.py

```diff
@@ -0,0 +1,67 @@
+"""Connects to the Cloud SQL database."""
+import os
+import MySQLdb
+
+
+class AirflowDB(object):
+  """"Provides the methods which allow interaction with the Airflow SQL database."""  # pylint: disable=line-too-long
+
+  def __init__(self, cloudsql_connection_name, cloudsql_user, cloudsql_password, cloudsql_host, cloudsql_db):
```

Comment:
please pass in a db connect as I act in the previous review.

Reply:
I commented on the last time you said this, and I was a little confused as to where you want me to place the db connect constructor? And since you want me to essentially move the entire init function out of the object, should I just remove the object completely and just have a simpler query function, or leave the db object, and initialize it using the db connection?

Reply:
Sorry I am just saying to pass the db connect in the constructor:
```python
def __init__(self, db_connect):
  self._db_connect = db_connect.
```

And then you create the database connection in your main. That way you can get rid of the env variables.